source/CIGameScene.cpp

```diff
@@ -119,22 +115,21 @@ bool GameScene::init(const std::shared_ptr<cugl::AssetManager>& assets,
     _opponent_planets.resize(5);
 
     // Game settings
-    _spawnRate = spawnRate;
-    _colorCount = colorCount;
+    _spawnRate = gameSettings->getSpawnRate();
+    _colorCount = gameSettings->getColorCount();
     
     // update stardustProb to match colorCount
-    for (int i = 0; i < colorCount; i++) {
+    for (int i = 0; i < _colorCount; i++) {
         _stardustProb[i] = 100;
     }
+    CIColor::setNumColors(_colorCount);
+    
+    _countdown = INACTIVE_WIN_COUNTER;
+    _timeElapsed = 0;
```

Comment:
what are these used for? or were they just missing inits?

Reply:
these were here before, I just moved them above the addChild lines

Reply:
I think we may want to organize the constants in these files. I know we talked about having a constants header before and I think it may be a good idea to get that implemented soon. I think a lot of the constants are confusing or repeated at this point.

Reply:
yeah totally agree. should probably make this a separate task at some point. or someone can just go in a do it when they have some free time